pkgs/os-specific/linux/kbd/search-paths.patch

```diff
@@ -52,10 +52,9 @@ Without this patch, kbd will only look inside
  };
 --- a/src/loadkeys.c
 +++ b/src/loadkeys.c
-@@ -27,6 +27,7 @@
+@@ -27,5 +27,6 @@
  
  static const char *const dirpath1[] = {
- 	"",
 +	"/etc/kbd/" KEYMAPDIR "/**",
```

Comment:
is this patch still needed? i don't see a difference in this line

https://github.com/legionus/kbd/compare/v2.6.1...v2.6.2#diff-c630bb579468ae349fabd8cd7733165012c1db7bf870061ba55095faeed1bf97R29

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/linux/kbd/search-paths.patch adds `/etc/kbd` search paths. I did not change that fact.

The change only adapts to the context change introduced by https://github.com/legionus/kbd/commit/a770f39f79e6a5fdf8f352d8b26d6a50c6b137ff

Does it make sense?

Reply:
I have mixed this up, sorry. Now it makes sense.